models/ac_generator.h

```diff
@@ -97,6 +112,8 @@ class ac_generator : public Node
     double phi_deg_; //!< Phase of sine current (0-360 deg)
 
     Parameters_(); //!< Sets default parameter values
+    Parameters_( const Parameters_& );
+    Parameters_& operator=( const Parameters_& p ); // Copy constructor EN
```

Comment:
The initials of the person who wrote the comment in 2009 which got here by copy-paste from another model (really looking forward to NESTML ...). The comment is also incorrect---this is the assignment operator, not the copy constructor. @appukuttan-shailesh You can just delete the comment, the code speaks for itself. Same applies in other models.